draft-ietf-webtrans-http3.md

```diff
@@ -191,7 +191,10 @@ be deemed incompatible, and MUST result in a H3_SETTINGS_ERROR connection
 error.
 
 The `webtransport` HTTP Upgrade Token uses the Capsule Protocol as defined in
-{{HTTP-DATAGRAM}}.
+{{HTTP-DATAGRAM}}.  The Capsule Protocol is negotiated when the server sends a
+2xx response.  The `capsule-protocol` header field {{Section 3.4 of
+HTTP-DATAGRAM}} is not used in WebTransport and MUST be ignored by WebTransport
```

Comment:
As written this discourages the use of the header. Here's a proposal that avoids that issue:

```suggestion
HTTP-DATAGRAM}} is not required by WebTransport and can safely be ignored by WebTransport
```